pyctuator/pyctuator.py

```diff
@@ -89,6 +89,9 @@ def __init__(
         self.boot_admin_registration_handler: Optional[BootAdminRegistrationHandler] = None
 
         root_logger = logging.getLogger()
+        if not root_logger.hasHandlers():
+            logging.info("Logging not configured, using logging.basicConfig()")
```

Comment:
seems like you rely on the call to `logging.info()` to implictly call `logging.basicConfig()`. I think it is clearer if you just call it yourself...

Reply:
We thought its a good idea to let the users know that because they didn't initialize the logging system, pyctuator did it for them, so its kind of "two birds with one stone".
However, calling `logging.basicConfig(...)` directly without logging is also good.

Reply:
I don't mind whether we log it or not, I just mind that we don't call basicConfig explicitly